test/flow_runner/spec/blocks/select_one_response_test.exs

```diff
@@ -0,0 +1,1078 @@
+defmodule FlowRunner.Spec.Blocks.SelectOneResponseTest do
+  use ExUnit.Case
+  alias FlowRunner.Spec.Blocks.SelectOneResponse
+
+  test "evaluate_outgoing/5 selects the correct outgoing link when the user input is a string resembling a boolean" do
+    flow = %FlowRunner.Spec.Flow{
```

Comment:
is there any chance we can use JSON to load this flow rather than including it verbatim in this test or is there a specific reason why we'd prefer doing that?

Here's an example of how I've used that before:

https://github.com/turnhub/flow_runner/blob/50a83d32d357f23937943bde176444f1dddd13e9/test/mega_test.exs#L3-L8

Reply:
great suggestion, done in https://github.com/turnhub/flow_runner/pull/180/commits/5de6d7d6b61ff61d806906d18d60c69e1075c995 and https://github.com/turnhub/flow_runner/pull/180/commits/44b6f7588457312d7d2da7a87520615ddf45bb60